rules/linux/execution_python_tty_shell.toml

```
@@ -10,32 +10,49 @@ Identifies when a terminal (tty) is spawned via Python. Attackers may upgrade a
 interactive tty after obtaining initial access to a host.
 """
 from = "now-9m"
-index = ["auditbeat-*", "logs-endpoint.events.*"]
-language = "kuery"
+index = ["logs-endpoint.events.*"]
+language = "eql"
 license = "Elastic License v2"
 name = "Interactive Terminal Spawned via Python"
 risk_score = 73
 rule_id = "d76b02ef-fc95-4001-9297-01cb7412232f"
 severity = "high"
 tags = ["Elastic", "Host", "Linux", "Threat Detection", "Execution"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and process.name:python and
-  process.args:("import pty; pty.spawn(\"/bin/sh\")" or
-                "import pty; pty.spawn(\"/bin/dash\")" or
-                "import pty; pty.spawn(\"/bin/bash\")")
+sequence with maxspan=5m
```

in order to make this more efficient, we can drop this down significantly since it would be an immediate spawn

 ```suggestion
sequence with maxspan=20s
```